code/modules/materials/material_sheets.dm

```diff
@@ -197,8 +197,8 @@
 /obj/item/stack/material/platinum/random
 	rand_min = 1
 	rand_max = 10
-	//spawn_tags = SPAWN_TAG_MATERIAL_RESOURCES_RARE
-	//rarity_value = 45
+	spawn_tags = SPAWN_TAG_MATERIAL_RESOURCES_RARE  //Occulus edit: Re-adding platinum glass to spawn pools
```

Comment:
Move this to the non-rare tag so science doesn't get cursed with a useless material at roundstart.